src/clj/harja/palvelin/integraatiot/api/analytiikka.clj

```diff
@@ -51,20 +51,29 @@
 (def db-reitti->avaimet
   {:f1 :reittipiste_aika
    :f2 :reittipiste_tehtavat
-   :f3 :reittipiste_sijainti
-   :f4 :reittipiste_materiaalit})
+   :f3 :reittipiste_sijainti_epsg4326
+   :f4 :reittipiste_sijainti_epsg3067
+   :f5 :reittipiste_materiaalit})
 
 (defn rakenna-reittipiste-sijainti
   "Reittipisteen sijainnin tiedot tulevat row_to_json funktion käytön vuoksi tekstimuodossa, joten
   niiden käsittely koordinaattimuotoon on monimutkaista."
-  [reitti]
-  (let [sijainti (:sijainti (:reittipiste reitti))
-        sijainnit (when sijainti (str/split sijainti #","))
-        koordinaatit (when sijainnit {:x (str/replace (first sijainnit) #"\(" "")
-                                      :y (str/replace (second sijainnit) #"\)" "")})
+  [reitti koordinaattimuutos?]
```

Comment:
Tätä voisi hiukan selventää.
Tässä taitaa olla kyse siitä, että otetaan mukaan reittiin myös muunnetut koordinaatit.
Mutta, koordinaatistomuunnos taidetaan silti aina tehdä SQL-puolella ja se tulee kyselyn mukana.
Olisi varmaan parempi nimetä "koordinaattimuunnos?" -> "lisaa-epsg-4326-koordinaatit?" tai vastaavaa, niin tietää että mitä oikein otetaan mukaan.

Lisäksi, kutsutaan turhaan suorityksen yhteydessä `(tee-koordinaatit (get-in reitti [:reittipiste :sijainti :epsg4326]),` sen tuloksia ei välttämättä oteta mukaan retittin joka kerta.

Reply:
Nämä ja slackissa puhutut muutokset tehty. Rajapintakutsussa käytetään edelleen parametrina `koordinaattimuutos`, koska siitä kerettiin jo puhua analytiikkaprojektin kanssa, ja se on mukavan lyhyt.